src/main/webapp/i18n/de/programmingExercise.json

```diff
@@ -329,7 +329,22 @@
                     "testcase": "Testfall",
                     "errorCategory": "Fehlerkategorie",
                     "totalItems": "Insgesamt {{count}} Elemente",
-                    "error": "Beim Laden des Feedbacks ist ein Fehler aufgetreten."
+                    "error": "Beim Laden des Feedbacks ist ein Fehler aufgetreten.",
+                    "search": "Suche ...",
+                    "feedbackModal": {
+                        "header": "Fehler Details",
+                        "feedbackTitle": "Test Fälle Feedback",
```

Comment:
We don't use spaces in German, we normally build compound words or at least use a hyphen (-).